flask_admin/templates/bootstrap3/admin/model/list.html

```diff
@@ -111,7 +111,7 @@
                             <input type="hidden" name="id" value="{{ get_pk_value(row) }}"/>
                             <input type="hidden" name="url" value="{{ return_url }}"/>
                             {% if delete_form.csrf_token %}
-                            <input name="csrf_token" type="hidden" value="{{ delete_form.csrf_token() }}"/>
+                            <input name="csrf_token" type="hidden" value="{{ delete_form.csrf_token._value() }}"/>
```

Comment:
Yeah you can, and it works with both Flask-WTF and WTForms 2. I'll submit the pull request to change it.